x-pack/plugins/security/public/components/management/users/confirm_delete.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component } from 'react';
+import {
+  EuiOverlayMask,
+  EuiConfirmModal,
+} from '@elastic/eui';
+import { toastNotifications } from 'ui/notify';
+export class ConfirmDelete extends Component {
+  deleteUsers = () => {
+    const { usersToDelete, apiClient, callback } = this.props;
+    const usernames = usersToDelete.map(user => user.username);
+    const errors = [];
+    usersToDelete.forEach(async (username) => {
+      try {
+        await apiClient.deleteUser(username);
+        toastNotifications.addSuccess(`Deleted user ${username}`);
+      } catch (e) {
+        errors.push(username);
+        toastNotifications.addDanger(`Error deleting user ${username}`);
+      }
+      if (callback) {
+        callback(usernames, errors);
+      }
+    });
+  };
+  render() {
+    const { usersToDelete, onCancel } = this.props;
+    const moreThanOne = usersToDelete.length > 1;
+    return (
+      <EuiOverlayMask>
+        <EuiConfirmModal
+          title={`Confirm delete user${moreThanOne ? 's' : ''}`}
+          onCancel={onCancel}
+          onConfirm={this.deleteUsers}
+          cancelButtonText="Cancel"
+          confirmButtonText="Delete"
+          buttonColor="danger"
+        >
+          <div>
+            <p>
+              You are about to delete {moreThanOne ? 'these' : 'this'} user{moreThanOne ? 's' : ''}:
```

Comment:
then this description and following list won't be necessary for deleting a single user